docs/UserGuide.md

```diff
@@ -98,10 +93,10 @@ In NetworkBook, details of a contact are categorised into different **fields**.
 
 Some fields are **single-valued fields**, meaning that each contact has **only 1 value** for each of these fields.
 
-| Single-valued fields | Prefix    | Format                                                       | Remarks                                                      |
-| -------------------- | --------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| name                 | /name     | Any non-empty value                                          | -                                                            |
-| priority level       | /priority | Either `high`, `medium` or `low`, or the initial alphabet    | Not case-sensitive                                           |
+| Single-valued fields | Prefix    | Format                                                                              | Remarks                                                  |
+| -------------------- | --------- |-------------------------------------------------------------------------------------| -------------------------------------------------------- |
+| name                 | /name     | Any non-empty value comprising alphanumeric characters (and spaces, if any)         | Is case-sensitive                                       |
```

Comment:
It is now not case sensitive. We have fixed this since it was indicated in UG by v1.3 that it is a fixable bug.